index.html

```diff
@@ -1,18 +1,62 @@
-<script>
-let name = document.location.pathname.slice(1);
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Yelpington</title>
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
+    />
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+    <style>
+      #map {
+        height: 400px;
+      }
+      #head {
+        background: green;
+        border-radius: 5px;
+      }
+      body {
+        background: rgb(91, 201, 91);
+      }
+    </style>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div id="header">Yelpington</div>
+    <div id="map"></div>
+    <script>
+      var map = L.map("map").setView([44.477, -73.2144], 17);
 
-if (!name) {
-  console.log("no place name specfied");
-} else {
-  console.log("fetching place named '" + name + "'");
+      L.tileLayer("https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png", {
+        attribution:
+          '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+      }).addTo(map);
+      var marker = L.marker([44.477, -73.2144]).addTo(map);
+      function onMapClick(e) {
+        alert("You clicked the map at " + e.latlng);
+      }
 
-  fetch(name + '.json')
-    .then(function (response) {
-      return response.text();
-    })
-    .then(function (myText) {
-      console.log(myText);
-    });
-}
-
-</script>
+      map.on("click", onMapClick);
+    </script>
+    <button id="getAll">Get All Restaurants</button>
+    <div id="all"></div>
+    <script>
+      document.getElementById("getAll").addEventListener("click", getAll);
+      function getAll() {
+        fetch("all.json")
+          .then(function(resp) {
+            return resp.json();
+          })
+          .then(function(mkJson) {
+            document.getElementById("all").innerHTML = JSON.stringify(mkJson);
```

Comment:
This just displays the raw .json file's contents to the page. We wanted a list of individual items you can access through that file (the restaurant names) formatted and displayed on the page. You've got a couple more steps to get there:
1. take each value individually. all.json contains an array so you can do this easily by using `.forEach()`

2. format the data. You can do this either by formatting the given data (e.g. split on dashes, and join with spaces), OR you can then fetch those individual restaurant's json files and get the `name` property off of them.

3. THEN you can display those names in some sort of list form on your index.html


-